### PR TITLE
Proof of membership

### PR DESCRIPTION
**Pull request by patmbolger**
PT Story: https://www.pivotaltracker.com/story/show/152839567

Changes proposed in this pull request:
1. Using IMGKit to convert HTML to image (jpeg)
2. Assemble proof-of-membership (POM) image using components including "app config" items (chairperson signature, SHF logo), member photo, and membership information. 
3. Allow member to download JPG image of POM, for use elsewhere
4. Also provide a HTML view of the POM by itself, in case member wants to take their own screenshot and use that instead (or in addition to) the JPG image, or, the member wants to use this link on their own website (instead of using the image directly - this will result in an always-updated POM to be created on demand).
5. Removed ENV guard that prevented the member from seeing the member photo upload form (in their profile-edit page).  Now the member sees that.

The member accesses their POM in the login-menu item "View Your Account".

Prerequisites to deployment:

1. Deploy to heroku and confirm it works there (completed, Wednesday, Feb 7)
2. Deploy to DO (staging) and confirm
3. In production, in "app configuration", add/update the chairperson signature and the SHF logo (this step can be done as part of the production deployment - images for this are included in this PR).

**Critical Deployment Steps**:

1. After the code is deployed, and _before_ the app is restarted, run:
```
rake shf:one_time:create_app_config
```
This will check whether an AppConfiguration instance exists, and create it if not.

2. Reload AppConfiguration images (SHF logo, Chairperson signature), using these files in `app/assets/images`:
  - chair_signature.png
  - shf_logo.png

These are compliant with the size requirements for these images (which have changed since AppConfiguration was first delivered).

Follow-on stories to be created in PT:

1. The JPG image has an 8 px margin, imposed by the user agent (browser).  This should be removed in the image, either through CSS or perhaps some image manipulation tool.

2. Adjust HTTP header to allow a member to include the HTML version on the website, within an iframe.

Screenshots (Optional):
### Member account view (zoomed out to see entire page):

screen shot 2018-02-03 at 7 01 21 am

### Result of clicking "Show Image" in prior page (actual size):

screen shot 2018-02-03 at 7 01 59 am

### Image produced by "Download Image" button:

screen shot 2018-02-03 at 7 02 37 am

Ready for review:
@AgileVentures/shf-project-team

### CODE REVIEW

**Comment by patmbolger** [comment]
**Update Feb 5**:  Allow member-info area to expand as needed to accommodate any number of categories:

screen shot 2018-02-05 at 9 24 40 am

File: config/locales/sv.yml [hunk 1]
```diff
@@ -632,6 +632,16 @@ sv:
       member: *member
       pay_membership_tooltip: När du betalar din medlemsavgift förlängs ditt medlemskap i ett år.
       membership_expire_date_tooltip: Efter detta datum går perioden för ditt medlemskap ut.
+      proof_of_membership: Bevis på medlemskap
```
Comment (thesuss) [inline config/locales/sv.yml]: Behörighetsbevis

File: config/locales/sv.yml [hunk 2]
```diff
@@ -632,6 +632,16 @@ sv:
...
+      proof_of_membership_describe: Den här bilden är ditt bevis på medlemskap i SHF.
```
Comment (thesuss) [inline config/locales/sv.yml]: /* Remember we are moving away from the use of "SHF" */ 

Detta är ditt behörighetsbevis.

File: config/locales/sv.yml [hunk 3]
```diff
@@ -632,6 +632,16 @@ sv:
...
+      proof_of_membership_image: Du kan ladda ner den här bilden och använda den
+                                 som du vill, inklusive på din webbplats, eller
+                                 tryckt ut och visas på din verksamhetsort.
```
Comment (thesuss) [inline config/locales/sv.yml]: tryckt ut och visas på din verksamhetsort. = 
skriva ut den och visa upp fysiskt i möten med kunder.

File: config/locales/sv.yml [hunk 4]
```diff
@@ -632,6 +632,16 @@ sv:
...
+      proof_of_membership_show: Eller du kan ta en skärmdump av bilden och använda
```
Comment (thesuss) [inline config/locales/sv.yml]: Eller du kan ta = 
Du kan även ta

File: config/locales/sv.yml [hunk 5]
```diff
@@ -632,6 +632,16 @@ sv:
...
+                                den istället. Använd knappen "Visa bild" för att
+                                gå till en sida med bara den presenterade bilden.
```
Comment (thesuss) [inline config/locales/sv.yml]: med bara den presenterade bilden. = 
som bara innehåller bilden.

File: app/assets/stylesheets/proof-of-membership.css [hunk 6]
```diff
@@ -0,0 +1,60 @@
+.parent-div {
```
Comment (thesuss) [inline app/assets/stylesheets/proof-of-membership.css]: @import url('https://fonts.googleapis.com/css?family=Open+Sans');
body {
    font-family: 'Open Sans', sans-serif;
}

Comment (thesuss) [reply app/assets/stylesheets/proof-of-membership.css]: (above .parent-div, clearly)

**Review by thesuss — CHANGES_REQUESTED** [review]

**Review by thesuss — APPROVED** [review]

**Comment by patmbolger** [comment]
**Update Feb 12**

The number of files increased due to concurrent updates to specific gems in use, including
  - cucumber (2.4 > 3.1)
  - font-awesome (4.7 > 5.0)
  - rspec (3.6 > 3.7)

and others.

This caused many unrelated things to break.

**Pull request merged**